.github/workflows/merge.yml

```
@@ -12,8 +12,8 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, ubuntu-latest]
-        python-version: [3.7, 3.8, 3.9]
-        flavour: ['core', 'all', 'pmdarima', 'torch', 'fbprophet']
+        python-version: [3.7, 3.9]
```

```suggestion
        python-version: [3.7, 3.8, 3.9]
```

^ restoring 3.8 for merge workflow, as discussed

---

And we could then only test 3.9 for PRs maybe?